.github/workflows/vercel-preview-pr-target.yaml

```
@@ -0,0 +1,90 @@
+    types: [opened, synchronize, reopened, labeled]
+    paths-ignore:
+      - ".github/workflows/**"
+concurrency:
+  group: ${{ format('pr-{0}-vercel-preview', github.event.pull_request.number) }}
+  cancel-in-progress: true
+jobs:
+  Deploy-Preview-PR:
+    if: contains(github.event.pull_request.labels.*.name, 'vercel-preview')
+    runs-on: ubuntu-latest
+    permissions:
+       contents: read
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 7

(indentation)